src/index.md

```diff
@@ -8,6 +8,19 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+save your website's files locally, on your users' devices. Consider Workbox if
+you want to:
+
+* Make your site work offline.
+* Improve load performance. Even if you don't want to go fully-offline, you can
+  use Workbox to save and serve common files locally, rather than from the
+  network.
```

Comment:
Sure, I can do that

For my own understanding: Isn't there a use case for downloading resources before a user needs them? So even though the very first visit on the homepage (for example) will be constrained by a network, you could speed up the first time experience on whatever page the user visits next... I guess that's a repeat-visit experience, even though it's a first-time experience of that page for the user (this is all just semantics)